client/chrome-extension/src/contentScript/PageWatcher.js

```diff
@@ -76,11 +76,14 @@ class PageWatcher {
         // If it's not one of our events, then it
         // should be an object describing the state
         default:
-          console.log('Content script received: ', message);
-          if (message.is_playing) {
-            this.video.play(message.time);
-          } else {
-            this.video.pause(message.time);
+          console.debug('Content script received: ', message);
+          const { type, videoTime } = message;
+          switch (type) {
+            case 'PLAY':
+              this.video.play(videoTime);
+              return;
+            case 'PAUSE':
+              this.video.pause(videoTime);
```

Comment:
@samgqroberts does it matter that there's a `return;` above but not here?

Reply:
good eyes -- no it does not, only because no other statements would get executed in the function in this logic branch. in fact, the linter can tell this, and yells if we have the return here as a "useless return" :woman_shrugging: